### ISSUE TITLE
On the accuracy of training

### ISSUE THREAD

**Issue by krmui** [yoonkicho/PPLR#6]

**Comment by yoonkicho** [comment]
Hi,
Unsupervised reid training sometimes showed different performances each time, especially compared to supervised training, but it wasn't as serious as in your case. Could you share about your environment setting?
I think that, under pseudo-labels that changed with each epoch, some randomness, such as batch construction, can result in performance gaps.
We add a few lines on training codes to handle randomness, and you also can check our pretrained models from [link](https://drive.google.com/drive/folders/1m5wDOJG7qk62PjkoOpTspNmk0nhLc4Vi). I hope this helps with your reserach.
